.eslintrc.cjs

```diff
@@ -123,6 +123,7 @@ module.exports = {
 		"no-only-tests/no-only-tests": "error",
 
 		// These on-by-default rules don't work well for this repo and we like them off.
+		"@typescript-eslint/no-explicit-any": "off",
```

Comment:
I think you'll need to move this here https://github.com/KATT/tupleson/pull/21/files#diff-e1ce717e1179a0db14e90ec5374768a206651ca807db58352991eb7895ed7c9cR56 (line 56). That override is what extends the `plugin:@typescript-eslint/recommended-type-checked` config preset, which is what enables `@typescript-eslint/no-explicit-any`.